<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orchestrator</string>
  </property>
  <property name="toolTip">
   <string>Remove Selected Piece from List</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_5">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>List of Pieces</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QPushButton" name="clrPcListBtn">
          <property name="toolTip">
           <string>Clear the list</string>
          </property>
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QSplitter" name="splitter_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QTableWidget" name="pcTable">
           <property name="toolTip">
            <string>Descriptions of Pieces</string>
           </property>
           <property name="columnCount">
            <number>3</number>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Title</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Composer</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Performers</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="subPcBtn">
          <property name="toolTip">
           <string>Remove selected piece from the list</string>
          </property>
          <property name="text">
           <string>- Piece</string>
          </property>
         </widget>
        </item>
       </layout>
       <zorder>splitter_6</zorder>
       <zorder>splitter_6</zorder>
       <zorder>clrPcListBtn</zorder>
       <zorder>subPcBtn</zorder>
      </widget>
      <widget class="QSplitter" name="splitter_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QComboBox" name="resCombo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>5</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Calculation Result</string>
        </property>
       </widget>
       <widget class="QPushButton" name="calcBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Calculate the optimal schedule arrangement</string>
        </property>
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="addPcBtn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Piece Information</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Performers:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QSplitter" name="splitter_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Title :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="titleEdit">
           <property name="toolTip">
            <string>Title of the piece</string>
           </property>
           <property name="placeholderText">
            <string>Title</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_9">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Composer :</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="compEdit">
           <property name="toolTip">
            <string>Composer of the piece</string>
           </property>
           <property name="placeholderText">
            <string>Composer</string>
           </property>
          </widget>
         </widget>
         <widget class="QSplitter" name="splitter_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QListWidget" name="perfList">
           <property name="toolTip">
            <string>List of Performers</string>
           </property>
          </widget>
          <widget class="QPushButton" name="subPerfBtn">
           <property name="toolTip">
            <string>Remove Selected Performer from List</string>
           </property>
           <property name="text">
            <string>- Performer</string>
           </property>
          </widget>
          <widget class="QSplitter" name="splitter_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <widget class="QLineEdit" name="perfEdit">
            <property name="placeholderText">
             <string>Performer</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
           </widget>
           <widget class="QPushButton" name="addPerfBtn">
            <property name="toolTip">
             <string>Add Performer Name to List</string>
            </property>
            <property name="text">
             <string>+ Performer</string>
            </property>
           </widget>
          </widget>
         </widget>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="clrPcBtn">
         <property name="toolTip">
          <string>Clear Beneath Information</string>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>titleEdit</zorder>
   <zorder>label_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>pcTable</zorder>
   <zorder>calcBtn</zorder>
   <zorder>resCombo</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton</zorder>
   <zorder>groupBox_2</zorder>
   <zorder>compEdit</zorder>
   <zorder>label_3</zorder>
   <zorder>splitter_5</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>668</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>clrPcBtn</tabstop>
  <tabstop>titleEdit</tabstop>
  <tabstop>compEdit</tabstop>
  <tabstop>perfList</tabstop>
  <tabstop>subPerfBtn</tabstop>
  <tabstop>perfEdit</tabstop>
  <tabstop>addPerfBtn</tabstop>
  <tabstop>addPcBtn</tabstop>
  <tabstop>pcTable</tabstop>
  <tabstop>subPcBtn</tabstop>
  <tabstop>clrPcListBtn</tabstop>
  <tabstop>resCombo</tabstop>
  <tabstop>calcBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
